source/server/listener_impl.cc

```diff
@@ -1090,6 +1103,15 @@ bool ListenerImpl::hasCompatibleAddress(const ListenerImpl& other) const {
 }
 
 bool ListenerImpl::hasDuplicatedAddress(const ListenerImpl& other) const {
+  // Skip the duplicate address check if this is the case of a listener update with new socket
+  // options.
+  if (Runtime::runtimeFeatureEnabled(
+          "envoy.reloadable_features.enable_update_listener_socket_options") &&
```

Comment:
Since this string is used in multiple places, can you define it once in some constexpr? There are a few others that are defined like that that you can look at.

Reply:
will do